tests/nlu/test_rule_based_annotator.py

```diff
@@ -20,9 +20,34 @@
     "directors": {
         "Tom Hanks": "tom hank",
     },
+    "genres": {
+        "Action": "action",
+        "Adventure": "adventure",
+        "Crime": "crime",
+    },
+    "title": {
+        "The Godfather": "the godfather",
+        "The Godfather: Part II": "the godfather part ii",
+        "Othello": "othello",
+        "The Lion King": "the lion king",
+    },
+    "keywords": {
+        "a birthday party": "a birthday party",
+        "action figure": "action figure",
+    },
 }
 
 
+def process_value(text: str) -> str:
+    """Returns a process value fixture."""
+    return text
```

Comment:
not sure how returning same text is a process? is this supposed to be a dummy function?

Reply:
Added mock to the name to be more explicit.